.gitignore

```diff
@@ -44,3 +44,6 @@ integration/test.sqlite
 integration/output/
 integration/output-postgres/
 integration/output-mongo/
+
+# Generated docs
+docs/api/
```

Comment:
Added this to avoid conflicts from generated stuffs. 

The situation is the same with the current setup, if someone make a change locally and publish docs, will be publishing local changes. So care must be taken when publishing docs locally until we love this to CI.

Reply:
Also majority of our contributors are not using forks, so even the generated "source code links" should be fine and pointing to this repo and not some fork.

Reply:
I guess this should be documented somewhere.